charts/internal/seed-controlplane/charts/alicloud-cloud-controller-manager/templates/deployment.yaml

```
@@ -16,10 +16,9 @@ spec:
       role: cloud-controller-manager
   template:
     metadata:
-{{- if .Values.podAnnotations }}
       annotations:
+        checksum/secret-cloud-provider-config: {{ include (print $.Template.BasePath "/secret-config.yaml") . | sha256sum }}
 {{ toYaml .Values.podAnnotations | indent 8 }}
```

Just to be safe, I guess this should stay, or?
```suggestion
{{- if .Values.podAnnotations }}
{{ toYaml .Values.podAnnotations | indent 8 }}
{{- end }}
```

---

agree.